lib/wasmex.ex

```diff
@@ -188,8 +188,13 @@ defmodule Wasmex do
       rescue
         e in RuntimeError -> {false, e.message}
       end
+      
+    params = case return_value do
```

Comment:
we are shadowing the `params` variable (from above) here and assign it a new use - handling return values.

We could call that variable `return_values` instead, what do you think?

```suggestion
    return_values = case return_value do
```

(also needs a change down in line 197)

Reply:
💯